tests/geobase/FairTestGeo.cxx

```diff
@@ -0,0 +1,16 @@
+/********************************************************************************
+ * Copyright (C) 2014-2024 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
```

Comment:
This file is very new… so probably should start with 2024?

And please update the copyright on all the files you changed.

Reply:
Done.